### PR TITLE
:art: Adding nhs.uk front-end library version 0.8.0

### PR DESCRIPTION
**Pull request by AdamChrimes**
## Description
Adding the NHS.UK front-end library (version 0.8.0) into the project 
https://github.com/nhsuk/frontend-library

## Checklist

- [ ] An [ADR](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions)
- [x] New and/or updated tests
- [x] Changes log in `CHANGELOG`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM node:8.9.1-alpine
 
-# RUN apk add --no-cache python=2.7.13-r1 git-perl bash make gcc g++
-# RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+RUN apk add --no-cache python==2.7.14-r0 git-perl bash make gcc g++
```
Comment (st3v3nhunt) [inline Dockerfile]: Is there supposed to be a double equals here?

File: app/views/includes/foot.nunjucks [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<script src="{{ SITE_ROOT }}/js/vendor/jquery-1.12.4.min.js"></script>
+<script src="{{ SITE_ROOT }}/js/app.js"></script>
```
Comment (st3v3nhunt) [inline app/views/includes/foot.nunjucks]: This file doesn't exist atm and is causing 404s on the front end, can it be removed please?

File: Dockerfile (hunk 1 above)
Comment (markysoft) [reply Dockerfile]: no, good spot.

File: app/views/layout.nunjucks [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        a.appendChild(r);
+        })(window,document,'//static.hotjar.com/c/hotjar-','.js?sv=');
+      </script>
+    {% endif %}
+
+    {% block meta %}{% endblock %}
+  </head>
+
+  <body class="{% block bodyClass %}{% endblock %}">
+    <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
```
Comment (st3v3nhunt) [inline app/views/layout.nunjucks]: This is causing a Content Security Policy error to be flagged. Check the JS console in the browser when loading the page.

File: app/views/layout.nunjucks [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    <!-- <script>(function(d){d.className=d.className.replace(/\bno-js\b/,'js-enabled')})(document.documentElement);</script> -->
```
Comment (st3v3nhunt) [inline app/views/layout.nunjucks]: Does this need to be in the source if it is commented out?

**Pull request merged**